File: dotnet/Pyrolite/Pyro/DummyPyroSerializer.cs
/* part of Pyrolite, by Irmen de Jong (irmen@razorvine.net) */
	
using System;
using System.Collections;

namespace Razorvine.Pyrolite.Pyro
{
	/// <summary>
	/// Dummy class to be able to unpickle Pyro Proxy objects.
	/// </summary>
	public class DummyPyroSerializer
	{
		/// <summary>
		/// for the unpickler to restore state
		/// </summary>
		public void __setstate__(Hashtable values) {
		}		
	}
}

File: java/src/main/java/net/razorvine/pyro/DummyPyroSerializer.java
package net.razorvine.pyro;

import java.io.IOException;

/**
 * Dummy class to be able to unpickle Pyro Proxy objects.
 * 
 * @author Irmen de Jong (irmen@razorvine.net)
 */
public class DummyPyroSerializer {
	/**
	 * called by the Unpickler to restore state
	 */
	public void __setstate__(java.util.HashMap<?,?> args) throws IOException {
	}
}
